__tests__/gds.test.ts

```diff
@@ -32,13 +32,15 @@ test('fetch artifacts', async () => {
 })
 
 test('errors when downloading artifacts', async () => {
-  await expect(downloadGraalVMEE('invalid', '22.1.0', '11')).rejects.toThrow(
+  await expect(
+    downloadGraalVMEELegacy('invalid', '22.1.0', '11')
+  ).rejects.toThrow(
     'The provided "gds-token" was rejected (reason: "Invalid download token", opc-request-id: /'
   )
-  await expect(downloadGraalVMEE('invalid', '1.0.0', '11')).rejects.toThrow(
-    'Unable to find JDK11-based GraalVM EE 1.0.0'
-  )
-  await expect(downloadGraalVMEE('invalid', '22.1.0', '1')).rejects.toThrow(
-    'Unable to find JDK1-based GraalVM EE 22.1.0'
-  )
+  await expect(
+    downloadGraalVMEELegacy('invalid', '1.0.0', '11')
+  ).rejects.toThrow('Unable to find JDK11-based GraalVM EE 1.0.0')
+  await expect(
+    downloadGraalVMEELegacy('invalid', '22.1.0', '1')
+  ).rejects.toThrow('Unable to find JDK1-based GraalVM EE 22.1.0')
```

Comment:
JDK1 > JDK11

Reply:
This is actually the error message and part of the test.